CodingStyle

```diff
@@ -155,8 +155,10 @@ For Angular code, we follow the official Angular style guide:
 
     https://angular.io/guide/styleguide
 
-To check whether your code is conformant with the style guide, we suggest
-using TSLint with Codelyzer:
+To check whether your code is conformant with the style guide, we use a
+combination of TSLint, Codelyzer and Prettier:
 
-    https://www.npmjs.com/package/codelyzer
+    https://palantir.github.io/tslint/
+    http://codelyzer.com/
```

Comment:
https://codelyzer.com/ works, but Firefox does not like the SSL certificate. Chrome 68 will warn you if you browse to a not SSL encrypted site.

Reply:
Codelyzer website doesn't have a valid SSL certificate.
Even they, on github [1], send you to the http site.

I think it's better to have chrome just showing a non intrusive warning, over all browsers preventing you from accessing the site before you accept the risk and add an exception to the browser configuration.

[1] https://github.com/mgechev/codelyzer